client/src/NetworkClient.ts

```diff
@@ -117,4 +213,47 @@ export class NetworkClient {
     public async removeTxFromMempool(txObj: PlainTransaction): Promise<void> {
         return this._eventClient.call('removeTxFromMempool', txObj);
     }
+
+    private _evictCachedTransactions() {
+        const CACHE_DURATION = 20;
```

Comment:
What is the reasoning for keeping the cache for 20 blocks, instead of, say, 60?

Reply:
In the end it's an arbitrary number. I however decided for a rather small number because I think usually one wouldn't query for mined/expired/relayed/pending transactions from long time ago.
I'll increase the number to 30.